website/docs/Use-Cases/Task-Oriented-AutoML.md

```diff
@@ -14,6 +14,7 @@
     - 'rank': learning to rank.
     - 'seq-classification': sequence classification.
     - 'seq-regression': sequence regression.
+    - 'multiplechoice-classification': multiple choice classification.
```

Comment:
This change could conflict with my PR. Could you check? If it does, remove this change.